jobs/mozillians_magic.rb

```diff
@@ -42,19 +41,22 @@ def query_mozillians(params = {})
       http.request(request)
     end
 
-    def query_mozillians_group(group)
-      response = query_mozillians({ group: group })
-
-      if response.code.to_i == 200
-        res = JSON.parse(response.body)
-        count = res['count']
+    def iterate_mozillians_groups(groups)
+      remove_from_group('mozillians_nda')
```

Comment:
> With this change you can avoid calling the method per group/multiple times. feel free to ignore this.

I won't change it, just to make it consistent with `add_to_group` below

> This way if ever need to change the name, you can do so only in one place instead of changing multiple files.

It's only within this one function we ever need to have access to the group names, and defining them as a variable would make the logic below rather complicated.